lib/ansible/modules/cloud/amazon/cloudformation.py

```diff
@@ -93,6 +93,13 @@
         present, the stack does exist, and neither 'template' nor 'template_url' are specified, the previous template will be reused.
     required: false
     version_added: "2.0"
+  create_changeset:
+    description:
+      - If stack already exists create a changeset instead of directly applying changes.
```

Comment:
This may seem obvious, but it'd be nice to add to this doc something like "WARNING: if the stack does not exist, it will be created. If the state is `absent`, the stack will be deleted immediately with no changeset." since I can see people missing that part.